### PR TITLE
Fix/traefik routes maintenance page and testing

### PR DESCRIPTION
**Pull request by mrnicegyu11**
- Adds traefik rules to obain certificates for `UUID.services.testing` routes (i.e. dynamic service container via testing routes)
- Fixes bug: Testing dynamic services are no longer captures by the maintenance page

Fixes git.speag.com/oSparc/osparc-ops-environments/-/issues/429

### CODE REVIEW

File: services/maintenance-page/docker-compose.yml.j2 [hunk 1]
```diff
@@ -26,7 +26,7 @@ services:
         - traefik.docker.network=${PUBLIC_NETWORK}
         - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.priority={{MAINTENANCE_PAGES_TRAEFIK_PRIORITY}}
         - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.rule={{  "Host(`" + j2item + "`)" }}
-        - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.rule=(Host(`{{j2item}}`) && PathPrefix(`/`)) || (HostRegexp(`services.{{j2item}}`,`{subhost:[a-zA-Z0-9-]+}.services.{{j2item}}`) && PathPrefix(`/`)) || (HostRegexp(`services.testing.{{j2item}}`,`{subhost:[a-zA-Z0-9-]+}.services.testing.{{j2item}}`) && PathPrefix(`/`))
+        - traefik.http.routers.{{"maintenance_page_web_" + j2item.replace('@','').replace(' ','').replace('.','').replace('-','').replace('\'','') + "_html"}}.rule=(Host(`{{j2item}}`) && PathPrefix(`/`)) || (HostRegexp(`services.{{j2item}}`,`{subhost:[a-zA-Z0-9-]+}.services.{{j2item}}`) && PathPrefix(`/`))
```
Comment (pcrespov) [inline services/maintenance-page/docker-compose.yml.j2:29]: do you have some sort of test for theses routes?

Comment (mrnicegyu11) [reply services/maintenance-page/docker-compose.yml.j2:29]: no, neither for the j2 nor for the routes. if you have smart suggestions for how to test this let me know :) thanks!

Comment (pcrespov) [reply services/maintenance-page/docker-compose.yml.j2:29]: I guess an obvious test is to run replacement against some samples that failed in the past
We do that with regex all the time for instance in https://github.com/ITISFoundation/osparc-simcore/blob/master/packages/models-library/tests/test_basic_regex.py


If you want to go further, another possibility is to use hypothesis (property-based testing) to automatically produce data and find edge cases.

**Comment by mrnicegyu11** [comment]
will not do regression/unit tests for now since first some "framework" for testing would have to be built. 
merging this into master for manual testing, thanks for the input.

**Review by GitHK — APPROVED** [review]
👍

**Comment by mrnicegyu11** [comment]
works on master! hooray!
